.github/workflows/environment-infra-cd.yml

```diff
@@ -59,7 +59,7 @@
         - name: 'Deploy'
           run: |
             cd dev-infrastructure/
-            make region metrics-infra
```

Comment:
the metrics-infra target is now part of the region pipeline so calling it explicitely is not required anymore. i missed that in my previous PR. GH actions are hard to test :(